Source/Core/Common/ChunkFile.h

```diff
@@ -312,6 +312,7 @@ class PointerWrap
       break;
 
     case MODE_WRITE:
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
This will not work on Windows and doesn't actually resolve the warning.

Reply:
Sure. In order to clean up this, it may require significant rewrite of this file.